src/App.jsx

```diff
@@ -1,31 +1,54 @@
 import './App.scss';
+import { useState } from 'react';
+
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+function filterQuery(field, filterValue) {
+  return [...field].filter(movie => {
```

Comment:
```suggestion
  return field.filter(movie => {
```
You don't need it here because the filter method does not mutate the array